src/pybkgmodel/camera.py

```diff
@@ -375,7 +375,7 @@ def get_pixel_coords(self):
 
         x = (self.xedges[1:] + self.xedges[:-1]) / 2
         y = (self.yedges[1:] + self.yedges[:-1]) / 2
-        xx, yy = np.meshgrid(x, y, indexing='ij')
+        xx, yy = numpy.meshgrid(x, y, indexing='ij')
```

Comment:
I think this is the wrong resolution of a merge conflict? 

The import at the top is `import numpy as np`